benchmarks/benches/lib.rs

```diff
@@ -452,30 +486,163 @@ fn successive_or(c: &mut Criterion) {
     group.finish();
 }
 
+// LEGACY BENCHMARKS
+// =================
```

Comment:
Do we really want to keep them? Isn't this covered by other benchmarks and tests?

Reply:
These are not helpful, but we should have _some_ meaningful way to measure `insert`, `remove`, `delete`, `insert_range` and `remove range`